app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -89,7 +89,7 @@ def custom_index_relation(collection)
     param :ostree_branch_id, String, :desc => N_("Id of an ostree branch to find repositories that contain that branch")
     param :library, :bool, :desc => N_("show repositories in Library and the default content view")
     param :archived, :bool, :desc => N_("show archived repositories")
-    param :content_type, RepositoryTypeManager.repository_types.keys, :desc => N_("limit to only repositories of this type")
+    param :content_type, RepositoryTypeManager.usable_repository_types.keys, :desc => N_("limit to only repositories of this type")
```

Comment:
This controls apipie generation which means that the apipie cache might be out of date, especially if the capabilities were updated.  Couple of ideas:

1) use repository_types and advertise all types to hammer
2) use repository_types only if usable_repository_types is empty
3) something else?

Reply:
I think using `defined_repository_types` might be fine.

Reply:
I think you broke nightlies ;)

https://community.theforeman.org/t/katello-nightly-rpm-pipeline-1033-failed/24460/4